### PR TITLE
tentative wording about 1.5.1 database roles

### PR DESCRIPTION
**Pull request by gaktive**

### CODE REVIEW

**Comment by gaktive** [comment]
Need more technical verbage here.

File: adoc/Release-Notes.adoc [hunk 1]
```diff
@@ -107,6 +107,8 @@ endif::[]
 [id='sec.1_5_1.issue']
 ==== Known Issues
 
+* Based on problems spotted in HA deployments and upgrades, it is recommended that you start any internal database role with a count of 1 first and then scale up to the needed number. This does not impact usage when connected to external databases.
```
Comment (satadruroy) [inline adoc/Release-Notes.adoc]: So far, we have noticed this problem only during HA deployments but better to provide instructions for starting with 1 PXC instance for both deployments and upgrades.

Comment (gaktive) [reply adoc/Release-Notes.adoc]: @btat should we update the docs first to reflect this and have the revised release notes point there?

Comment (gaktive) [reply adoc/Release-Notes.adoc]: We may have to be precise in what the problem is with what happens with multiple instances.

Comment (satadruroy) [reply adoc/Release-Notes.adoc]: The problem is captured, albeit very briefly, in the next paragraph (?):
"On occasion, the `uaa` pod may fail to start due to database migration failures." We should add to that : " `uaa` and possibly some other roles may fail to start due to database migration failures."

Comment (btat) [reply adoc/Release-Notes.adoc]: > 
> 
> @btat should we update the docs first to reflect this and have the revised release notes point there?

For upgrades, should I add back the different upgrade scenarios similar to 1.5 (https://documentation.suse.com/suse-cap/1.5/single-html/cap-guides/#sec-cap-update)?

For deployments, the docs always had initial steps as SA to provide a kind of minimal deployment. In the separate chapter for HA, we listed a prerequisite of starting with SA mysql instances before transitioning to HA and that prereq wasn't removed for 1.5.1.

Comment (satadruroy) [reply adoc/Release-Notes.adoc]: @btat yes, let's add back the upgrade scenarios similar to 1.4.1 to 1.5.

Comment (btat) [reply adoc/Release-Notes.adoc]: PR for docs made.

For the general upgrade section the link is https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-update

Link (once published) for clusters currently using `config.HA` will be https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-upgrade-config-ha

Link (once published) for clusters currently using custom sizing will be https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-upgrade-custom-sizing

Link (once published) for clusters in SA will be https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-upgrade-no-ha

Comment (gaktive) [reply adoc/Release-Notes.adoc]: Thanks @btat. Once the links are live, I'll attempt an initial wording with links to those sections.

**Comment by gaktive** [comment]
@btat keep me posted on the link status
@satadruroy would the 2nd point regarding uaa recovery still be needed? I suspect so.

**Comment by satadruroy** [comment]
> @satadruroy would the 2nd point regarding uaa recovery still be needed? I suspect so.
Yes. Even with a single instance of PXC you might still run into a UAA DB migration failure.

File: adoc/Release-Notes.adoc [hunk 2]
```diff
@@ -107,7 +107,20 @@ endif::[]
 [id='sec.1_5_1.issue']
 ==== Known Issues
 
-* On occasion, the `uaa` pod may fail to start due to database migration failures. In these situations, manual intervention is required to track the last completed transaction in the `uaadb` database, update the `schema_version` table with the record of the last completed transaction, then restart the migration. Please contact support for further instructions.
+[IMPORTANT] 
+====
+* For upgrades from {product} 1.5 to 1.5.1, if the `mysql` roles of `uaa` and `scf` are in high availability mode, it is recommended to first scale them to single availability. Performing an upgrade with these roles in HA mode runs the risk of encountering potential database migration failures, which will lead to pods not starting. This does not impact usage when connected to external databases.
```
Comment (troytop) [inline adoc/Release-Notes.adoc]: For clusters in a high availability (HA) configuration, the `mysql` roles of `uaa` and `scf` should be scaled down to single instances before upgrading. Upgrading with multiple instances of `mysql` running may cause database migration failures, which will lead to pods not starting after upgrade. Clusters using external databases (via `DB_EXTERNAL_*` settings) are unaffected.

**Review by troytop — CHANGES_REQUESTED** [review]
Maybe use this wording.

File: adoc/Release-Notes.adoc (hunk 2 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc]: Changed, thanks!

**Pull request merged**